src/main/resources/application.yml

```diff
@@ -53,6 +53,7 @@ wave:
   build:
     buildkit-image: "moby/buildkit:v0.25.2-rootless"
```

Comment:
For the remaining buildkit image it feels like we should align and mirror and serve from `public.cr.seqera.io/wave/buildkit` also.